From f9639bd6e1d920cc46ff56ec11981536cb371c6b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hubert=20Nod=C5=BCak-Pluta?=
 <hubertnodzak-pluta@MBP-infakt-8.localdomain>
Date: Fri, 27 Sep 2019 12:17:48 +0200
Subject: [PATCH] correct chmods - using too high causes setting 777 during gem
 install that's insecure on production env.

---
 .gitignore           | 0
 LICENSE              | 0
 README.md            | 0
 lib/eui48.rb         | 0
 lib/eui64.rb         | 0
 lib/ipv4.rb          | 0
 lib/ipv4net.rb       | 0
 lib/ipv6.rb          | 0
 lib/ipv6net.rb       | 0
 lib/mask128.rb       | 0
 lib/mask32.rb        | 0
 lib/netaddr.rb       | 0
 lib/util.rb          | 0
 netaddr.gemspec      | 0
 test/eui48_test.rb   | 0
 test/eui64_test.rb   | 0
 test/examples.rb     | 0
 test/ipv4_test.rb    | 0
 test/ipv4net_test.rb | 0
 test/ipv6_test.rb    | 0
 test/ipv6net_test.rb | 0
 test/mask128_test.rb | 0
 test/mask32_test.rb  | 0
 test/netaddr_test.rb | 0
 test/run_all.rb      | 0
 25 files changed, 0 insertions(+), 0 deletions(-)
 mode change 100755 => 100644 .gitignore
 mode change 100755 => 100644 LICENSE
 mode change 100755 => 100644 README.md
 mode change 100755 => 100644 lib/eui48.rb
 mode change 100755 => 100644 lib/eui64.rb
 mode change 100755 => 100644 lib/ipv4.rb
 mode change 100755 => 100644 lib/ipv4net.rb
 mode change 100755 => 100644 lib/ipv6.rb
 mode change 100755 => 100644 lib/ipv6net.rb
 mode change 100755 => 100644 lib/mask128.rb
 mode change 100755 => 100644 lib/mask32.rb
 mode change 100755 => 100644 lib/netaddr.rb
 mode change 100755 => 100644 lib/util.rb
 mode change 100755 => 100644 netaddr.gemspec
 mode change 100755 => 100644 test/eui48_test.rb
 mode change 100755 => 100644 test/eui64_test.rb
 mode change 100755 => 100644 test/examples.rb
 mode change 100755 => 100644 test/ipv4_test.rb
 mode change 100755 => 100644 test/ipv4net_test.rb
 mode change 100755 => 100644 test/ipv6_test.rb
 mode change 100755 => 100644 test/ipv6net_test.rb
 mode change 100755 => 100644 test/mask128_test.rb
 mode change 100755 => 100644 test/mask32_test.rb
 mode change 100755 => 100644 test/netaddr_test.rb
 mode change 100755 => 100644 test/run_all.rb

diff --git a/.gitignore b/.gitignore
old mode 100755
new mode 100644
diff --git a/LICENSE b/LICENSE
old mode 100755
new mode 100644
diff --git a/README.md b/README.md
old mode 100755
new mode 100644
diff --git a/lib/eui48.rb b/lib/eui48.rb
old mode 100755
new mode 100644
diff --git a/lib/eui64.rb b/lib/eui64.rb
old mode 100755
new mode 100644
diff --git a/lib/ipv4.rb b/lib/ipv4.rb
old mode 100755
new mode 100644
diff --git a/lib/ipv4net.rb b/lib/ipv4net.rb
old mode 100755
new mode 100644
diff --git a/lib/ipv6.rb b/lib/ipv6.rb
old mode 100755
new mode 100644
diff --git a/lib/ipv6net.rb b/lib/ipv6net.rb
old mode 100755
new mode 100644
diff --git a/lib/mask128.rb b/lib/mask128.rb
old mode 100755
new mode 100644
diff --git a/lib/mask32.rb b/lib/mask32.rb
old mode 100755
new mode 100644
diff --git a/lib/netaddr.rb b/lib/netaddr.rb
old mode 100755
new mode 100644
diff --git a/lib/util.rb b/lib/util.rb
old mode 100755
new mode 100644
diff --git a/netaddr.gemspec b/netaddr.gemspec
old mode 100755
new mode 100644
diff --git a/test/eui48_test.rb b/test/eui48_test.rb
old mode 100755
new mode 100644
diff --git a/test/eui64_test.rb b/test/eui64_test.rb
old mode 100755
new mode 100644
diff --git a/test/examples.rb b/test/examples.rb
old mode 100755
new mode 100644
diff --git a/test/ipv4_test.rb b/test/ipv4_test.rb
old mode 100755
new mode 100644
diff --git a/test/ipv4net_test.rb b/test/ipv4net_test.rb
old mode 100755
new mode 100644
diff --git a/test/ipv6_test.rb b/test/ipv6_test.rb
old mode 100755
new mode 100644
diff --git a/test/ipv6net_test.rb b/test/ipv6net_test.rb
old mode 100755
new mode 100644
diff --git a/test/mask128_test.rb b/test/mask128_test.rb
old mode 100755
new mode 100644
diff --git a/test/mask32_test.rb b/test/mask32_test.rb
old mode 100755
new mode 100644
diff --git a/test/netaddr_test.rb b/test/netaddr_test.rb
old mode 100755
new mode 100644
diff --git a/test/run_all.rb b/test/run_all.rb
old mode 100755
new mode 100644
